### PR TITLE
fix: eslint warnings for v5

### PR DESCRIPTION
**Pull request by brunozoric**
## Related Issue
eslint warnings in v5- no defined issue

## Your solution
Unused variables were removed, imports were renamed and removed and some lines put under eslint disable (where really necessary).
For some function arguments a type was created with what it contains since it is not used but we want to know what is inside it.
There is some object destructuring when some property does not need (or must not) be used later on when passing that object and eslint is giving error about var not being used - here I set eslint-disable for that line. Maybe lodash omit can be used?

## How Has This Been Tested?
`yarn` -> `yarn build` -> `yarn webiny deploy api --env=local` -> `yarn start`

### CODE REVIEW

**Review by Ashu96 — COMMENTED** [review]
Looks good to me :+1: 
Nicely done @brunozoric  :zap: 

I really liked the simplified `scripts` inside `package.json`. For example,
`"eslint:fix": "yarn eslint --fix",`

File: package.json [hunk 1]
```diff
@@ -138,9 +138,9 @@
     "lint-staged": "lint-staged",
     "postinstall": "node ./scripts/linkPackages.js",
     "prepublishOnly": "node scripts/prepublishOnly.js",
-    "prettier": "prettier --config .prettierrc.js --write",
-    "prettier:check": "prettier \"**/**/*.{js,jsx,ts,tsx}\" --check --config .prettierrc.js",
-    "prettier:fix": "prettier \"**/**/*.{js,jsx,ts,tsx}\" --config .prettierrc.js --write",
+    "prettier": "prettier \"**/**/*.{js,jsx,ts,tsx}\" --config .prettierrc.js",
+    "prettier:check": "yarn prettier --check",
+    "prettier:fix": "yarn prettier --write",
```
Comment (doitadrian) [inline package.json:143]: Looks good, please just use the command here as well:
image

File: packages/project-utils/bundling/ssr/buildAppSSR.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ const fs = require("fs-extra");
 
 module.exports.buildAppSSR = async (
     { app = null, indexHtmlPath = null, createAppPath = null, output, ...options },
+    // eslint-disable-next-line
```
Comment (doitadrian) [inline packages/project-utils/bundling/ssr/buildAppSSR.js:6]: I think we can remove this line / the unused `context` below. I don't see it being used anywhere.

File: packages/project-utils/bundling/ssr/buildAppSSRFromSource.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ const { setupOutput } = require("./utils");
 
 module.exports.buildAppSSRFromSource = async (
     { handler = null, renderer = null, output = null, ...options },
+    // eslint-disable-next-line
```
Comment (doitadrian) [inline packages/project-utils/bundling/ssr/buildAppSSRFromSource.js:8]: Samo as `packages/project-utils/bundling/ssr/buildAppSSR.js`.

**Review by doitadrian — CHANGES_REQUESTED** [review]
Very nice cleanup 👏 

Just check the three super-tiny comments, and we merge this.

**Review by doitadrian — APPROVED** [review]

**Comment by doitadrian** [comment]
Awesome @brunozoric 👏

**Pull request merged**